### PR TITLE
Adds fix for pastie, relative timestamps

### PR DESCRIPTION
**Pull request by HashNuke**
- Fixes pastie
- Adds relative time stamps
- Refactors code to follow github's styleguide for js
- Centralizes settings in js

### CODE REVIEW

File: app/assets/javascripts/backbone/plugins/image_embed.js.coffee [hunk 1]
```diff
@@ -1,27 +1,29 @@
 class Kandan.Plugins.ImageEmbed
-  @image_regex:   /http.*\.(jpg|jpeg|gif|png)/i
+  @options:
+    regex:   /^http.*\.(jpg|jpeg|gif|png)/i
```
Comment (sgrove) [inline app/assets/javascripts/backbone/plugins/image_embed.js.coffee]: This is going to screw up the changes I made yesterday with the image embed plugin. This is after you merged master into your branch already?

Comment (HashNuke) [inline app/assets/javascripts/backbone/plugins/image_embed.js.coffee]: I merged master and went thru the conflicts and saw your changes. Preserved whatever was necessary and made edits to suit the @options pattern being used in the plugins now.

**Comment by HashNuke** [comment]
@sgrove fixed them. kandan seems to be down. hence the comment here.

**Comment by sgrove** [comment]
Working on bringing it back up.

On Thu, Apr 12, 2012 at 10:31 AM, Akash Manohar <
reply@reply.github.com

**Comment by sgrove** [comment]
back up

**Pull request merged**